buildspec.test.yml

```diff
@@ -6,6 +6,7 @@ phases:
       - echo Setting local Python versions
       - pyenv versions | awk 'match($0, /\d\.\d+\.\d+/) { print substr($0, RSTART, RLENGTH) }' > .python-version
       - echo Installing dependencies
+      - pip install --upgrade pip
```

Comment:
Why was this needed btw?

Reply:
This was my (failed) attempt to fix the failing build. Will remove in a sec